libtransmission/tr-udp.cc

```diff
@@ -109,6 +109,11 @@ void event_callback(evutil_socket_t s, [[maybe_unused]] short type, void* vsessi
             return;
         }
 
+        auto const from_str = [from_sa]
```

Comment:
Doesn't depend on anything inside the loop, could be moved out of `for (;;)`.

Reply:
d39fde3c4c5771573621d0ba620dcc58e409f49f